source/world_builder/objects/bezier_curve.cc

```diff
@@ -119,7 +132,7 @@ namespace WorldBuilder
             }
           }
 
-          for (size_t p_i = 1; p_i < n_points-1; ++p_i)
+          for (size_t p_i = 0; p_i < n_points-1; ++p_i)
```

Comment:
I think this is breaking the tests. Was this change intentional? If so, since this pull request is just about documentation changes, it would best go in a different pull request.